internal/cmd/kafka/command_mirror_list.go

```diff
@@ -26,10 +26,14 @@ func (c *mirrorCommand) newListCommand() *cobra.Command {
 				Text: `List all active mirror topics under "my-link":`,
 				Code: "confluent kafka mirror list --link my-link --mirror-status active",
 			},
+			examples.Example{
+				Text: "Omit `--link` to list all mirror topics in the cluster:",
```

Comment:
It would be great if each example could be standalone. For this set of examples that might look like:
1. List all active mirror topics.
2. List all active mirror topics for cluster link "my-link".

Reply:
Not quite sure what you mean, but I removed the first part of the text so that it doesn't look like they're related.